README.md

```diff
@@ -109,3 +113,44 @@ NEO.stats()
    .then(data => console.log(data))
    .catch(err => console.log(err));
 ```
+
+<h3 id="earth-imagery">Earth.imagery(options)</h3>
+
+Retrieves the Landsat 8 image for the supplied location and date
+
+##### `options` (optional) - **[Object]**
+* `lat` - **[Float]** The latitude of the imagery's location to retrieve.
+* `lon` - **[Float]** The longitude of the imagery's location to retrieve.
+* `date` - **[String]** The date of the imagery to retrieve. Default is today's date. Must be in the format `YYYY-MM-DD`
+* `cloud_score` - **[Boolean]** Cloud score calculates the percentage of the image covered by clouds when `true`. Default is `false`.
```

Comment:
@AJFunk I was getting errors passing any other values than the default value. I was thinking a separate issue should be created for debugging this property since going with the default value does not break anything.